applications/endoscopy_tool_tracking_distributed/cpp/README.md

```diff
@@ -21,13 +21,26 @@ If you want to manually convert the video data, please refer to the instructions
 
 ### Run Instructions
 
-Run the following command to start the application.  This will run the app with a pre-recorded video as input:
 
 ```sh
+# Build the Holohub container for the Distributed Endoscopy Tool Tracking application
+./dev_container build --docker_file applications/endoscopy_tool_tracking_distributed/Dockerfile
+
+# Launch the container
+./dev_container launch
+
+# Build the Distributed Endoscopy Tool Tracking application
+./run build endoscopy_tool_tracking_distributed
+
+# Generate the onnx engine
+python3 utilities/generate_trt_engine.py --input data/endoscopy/tool_loc_convlstm.onnx --output data/endoscopy/engines/ --fp16
+
 # Start the application with all three fragments
-./dev_container build_and_run endoscopy_tool_tracking_distributed --language cpp
+./run launch endoscopy_tool_tracking_distributed cpp
+
+# Once you have completed the above command, you may exit the container and
+#  use the following commands to run the application in distributed mode:
```

Comment:
Is running with all three fragments required in advance? Or just TRT engine generation?